back/engines/commercial/bulk_import_ideas/app/services/bulk_import_ideas/parsers/idea_pdf_file_parser.rb

```diff
@@ -22,9 +22,14 @@ def create_files(file_content)
         # Get number of pages in a form from the download
         pages_per_idea = import_form_data[:page_count]
 
-        pdf = ::CombinePDF.parse URI.open(source_file.file_content_url).read
+        pdf = begin
+          ::CombinePDF.parse URI.open(source_file.file_content_url).read
+        rescue ::CombinePDF::ParsingError
+          raise BulkImportIdeas::Error.new 'bulk_import_malformed_pdf', value: source_file.file_content_url
```

Comment:
Is `value` used later?

Reply:
No not now - but I left it in as I think it's useful to be able to see in the response.